scipy/stats/tests/test_stats.py

```diff
@@ -375,16 +375,25 @@ def test_length_two_neg2(self):
         assert_equal(r, -1)
         assert_equal(p, 1)
 
-    def test_more_basic_examples(self):
+    # Expected values computed with R 3.6.2 cor.test, except for r and the
+    # p-value for alternative='two-sided', which were computed with mpmath.
+    @pytest.mark.parametrize('alternative, pval, rlow, rhigh',
+                             [('two-sided',
+                               0.325800137536758, -0.814939, 0.992307),
+                              ('less',
+                               0.8370999, -1, 0.9856009),
+                              ('greater',
+                               0.1629001, -0.6785654, 1)])
+    def test_basic_example(self, alternative, pval, rlow, rhigh):
         x = [1, 2, 3, 4]
         y = [0, 1, 0.5, 1]
-        r, p = stats.pearsonr(x, y)
-
-        # The expected values were computed using mpmath with 80 digits
-        # of precision.
-        assert_allclose(r, 0.674199862463242)
-        assert_allclose(p, 0.325800137536758)
+        result = stats.pearsonr(x, y, alternative=alternative)
+        assert_allclose(result.r, 0.674199862463242, rtol=1e-12)
```

Comment:
```suggestion
        assert_allclose(result.r, 0.6741998624632421, rtol=1e-12)
```
Generated from code above.